core/domain/feedback_services_test.py

```diff
@@ -134,7 +134,7 @@ def setUp(self):
         # Closed thread without suggestion.
         thread4 = feedback_models.FeedbackThreadModel(
             id=feedback_models.FeedbackThreadModel.generate_full_thread_id(
-                self.EXP_ID2, self.THREAD_ID4),
+                self.EXP_ID1, self.THREAD_ID4),
```

Comment:
So, I've answered below to this note. I will include relevant parts here:

fake get_threads(i.e self._get_threads) returns: 
`
['exp1.thread1', 'exp1.thread2', 'exp1.thread3', 'exp2.thread4', 'exp2.thread4']. 
`
After that we try to call:
`
feedback_services.get_all_threads('exp2')
`
Which, in a consequence calls get_threads('exp2') which returns fake get_threads.
In a result, we will receive all threads including "exp1.\*", but we shouldn't.
Thus, I think this is logically wrong.
That is why I've changed _get_threads to return only "exp1.\*" and tests to use get_all_threads('exp1').